purchases/src/main/kotlin/com/revenuecat/purchases/common/events/BackendEvent.kt

```diff
@@ -44,10 +45,12 @@ internal sealed class BackendEvent : Event {
         @SerialName("dark_mode")
         val darkMode: Boolean,
         val locale: String,
-        @SerialName("is_sandbox")
-        val isSandbox: Boolean,
         @SerialName("display_mode")
         val displayMode: CustomerCenterDisplayMode,
+        // only valid for survey option chosen
+        val path: CustomerCenterConfigData.HelpPath.PathType?,
+        val url: String?,
```

Comment:
```        
        case surveyOptionID = "surveyOptionId"
        case surveyOptionTitleKey = "surveyOptionTitleKey"
        case additionalContext = "additionalContext"
```

Those are also missing

Reply:
Just checking but what does `additionalContext` is supposed to contain? Just wondering since I noticed we didn't add it yet.

Reply:
It's for when we add a free text option. It is not required to add it right now since it will be always null, but it's in the event definition.